bot/src/db.js

```diff
@@ -8,11 +8,11 @@ const etcd = new Etcd3({hosts: ETCD_URLS});
  * @returns {Promise<string[]>}
  */
 function listApps() {
-    return etcd.getAll().prefix("/clients").keys().then((apps) => {
-        for (let i = 0; i < apps.length; i++) {
-            let strs = apps[i].split("/", 3);
-            apps[i] = strs[strs.length - 1];
-        }
+    return etcd.getAll().prefix("/clients").keys().then((appss) => {
+        let apps = appss.map((sub) => { 
+            let res = sub.split("/"); 
+            return res[2] 
```

Comment:
@gutorc92, what happens if `res` length if less than `3`? I think using `sub.split("/", 3)` and returning `res[res.length - 1]` is safer.